autorag/nodes/generator/exo_openailike_llm.py

```diff
@@ -0,0 +1,215 @@
+import logging
```

Comment:
Can you tell me why you made `exo_openailike_llm`??
It is better just use LlamaIndexLLM `openailike`??
Plus, since we add chat history at prompt maker, it has no effect to the generator. And we can implement chat history without tackling with generator modules.

Reply:
Hey folks, some of this wasn't meant to be pushed yet - we're still testing! We'll roll back the PR, but implement the above comments. 

FYI LLamaIndexLLM openailike seems to not play well with Exo LLM endpoints due to something about the messages object passing. Re-writing the OpenAI lib to interact solved this issue, to a certain degree. 

Cheers,
Daniel & Team